hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java

```diff
@@ -254,4 +440,34 @@ public static void addKeyTableCacheEntries(
           new CacheValue<>(keyInfo, index));
     }
   }
+
+  /**
+   * Adding directory info to the Table cache.
+   *
+   * @param omMetadataManager  OM Metdata Manager
+   * @param dirInfo            directory info
+   * @param missingParentInfos list of the parents to be added to DB
+   * @param trxnLogIndex       transaction log index
+   */
+  public static void addDirectoryTableCacheEntries(
+          OMMetadataManager omMetadataManager,
+          Optional<OmDirectoryInfo> dirInfo,
+          Optional<List<OmDirectoryInfo>> missingParentInfos,
+          long trxnLogIndex) {
+    for (OmDirectoryInfo parentInfo : missingParentInfos.get()) {
+      omMetadataManager.getDirectoryTable().addCacheEntry(
+              new CacheKey<>(omMetadataManager.getOzoneLeafNodeKey(
```

Comment:
Here this should be in new format right?

like parentObjectID/name, as we use the format of parentObjectID/name when writing to DB in Response class.

Reply:
Yes, it is in the new format:   "parentInfo.getParentObjectID(), parentInfo.getName()".

I hope the variable name is confusing.

I think, rename variable 'parentInfo' to 'subDirInfo' , will make it more clear "subDirInfo.getParentObjectID(), subDirInfo.getName()".

Does it make sense to you?